### DISCUSSION TITLE
아카이브

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/9th-builder#6] [Show and tell]
쨘!

**Comment by seriousran** [comment]
현재 여러분들의 성장을 가로막는 벽은 무엇인가요?

**Reply by WongiPark0628** [reply]
스스로 어떤 행위를 하고자 할때 동시에 나타나는 두려움이 저에게는 벽인 것 같습니다..! 요번 9기 활동을 하면서 조금 더 도전적인 행동을 많이 해보고자 다짐합니다.

**Reply by suhyeong10** [reply]
저는 반복되는 업무가 벽인 것 같습니다! 얼마 전까지만 해도 항상 새로운 작업을 하고 그로 인해 새롭게 공부하며 성장할 수 있었는데, 현재는 반복적인 일만하다보니 그런 기회가 적어진 것 같습니다 ㅠㅠ 

이번 9기 활동을 하면서 새로운 경험을 많이 해보고 많은 성장을 이뤄보고자 합니다!

**Reply by thrcle** [reply]
저의 성장을 가로막는 것은 좁은 시야였습니다.

첫번째로 ‘나는 아직 주니어니까, 배울 점 많은 누군가가 나를 이끌어줬으면 좋겠다’라고 스스로 역할을 한정지었습니다. 
그러나 그러한 이상적인 상황은 오지 않았고, 마음이 오히려 성장의 걸림돌이 되었던 것 같습니다.

두번째로 내가 관심이 있고 열정을 쏟은 일에 자부심을 느끼기보단 남들의 관심과 시선에 흔들렸던 점입니다. 

이번 활동을 통해 가짜연구소에서 저의 성장을 가로막았던 벽을 허물어보려합니다.
잘 부탁드립니다. 감사합니다!

**Reply by shstat1729** [reply]
가짜연구소 9기 빌더 온보딩 과정을 하고 있습니다 ([Chanran Kim](https://www.linkedin.com/in/chanran-kim/) 진행). 1주차 내용은 가짜연구소 활동을 왜 하는지에 대해서 생각해보는 것입니다. 그래서 제가 가짜연구소 활동을 하는 이유와 학위과정을 하는 이유에 대해 고민해보았습니다. 그리고 성장을 가로막는 벽에 대해서도 고민해보았습니다.

1. 가짜연구소 활동을 하는 이유

 저는 규제 관련 업무를 하고 싶습니다. 환자에게 도움이 되는 좋은 의료기기와 의약제품이 출시되는 데에 기여를 하고 싶습니다. 그러기 위해서는 규제관련 가이드라인을 익혀야 합니다. 가짜연구소 활동을 하면서 규제관련 가이드라인을 꾸준히 익히고자 합니다. 7기에서는 임상시험 통계 관련 가이드라인을 익혔고, 8기에서는 real world evidence 확보에 중요한 인과추론을 공부하였습니다. 9기에서는 real world data / evidence 관련 FDA, MHRA, KFDA, ICH 가이드라인을 익혀보고자 합니다. 더 나아가 함께 지식을 익히고, 다양한 사람들이 공부할 수 있도록 자료를 공유하고자 합니다. 그렇게 함으로써 한국의 규제 과학 관련 역량이 높아질 것으로 기대합니다. 더 나아가 regulatory guideline을 만드는 데에도 기여하고 싶습니다. 규제 과학을 제대로 정립해야 환자에게 유익할 것이기 때문입니다.

 또, 전문적인 지식 함양 뿐만 아니라 가짜연구소 활동을 하면 네트워크를 구축하는 장점이 있습니다. 같은 분야의 실무자와 교류를 함으로써 많은 점들을 배울 수 있습니다.

내가 학위과정을 하는 이유는 무엇인가?

주도적으로 연구를 하고 싶어서 학위를 시작하였습니다. 석사과정에서 통계학을 전공했기 때문에 데이터를 분석하는 데 어려움은 크게 없었으나, 왜 이 데이터를 특정한 실험 디자인에 적용해야 하는지 잘 이해하지 못했습니다. 제가 스스로 연구 디자인을 세우고 연구를 하고 싶었습니다. 그러던 와중에 임상연구설계평가학과에서는 임상연구 디자인에 특화되어 있다는 것을 알게 되었습니다. 그래서 이 과정을 시작하게 되었습니다.

이곳에서의 학위과정에서 다양한 데이터를 분석할 수 있습니다. 예를 들어, 삼성서울병원의 electronic health record, 국민건강보험공단 청구 데이터를 분석할 수 있지요. 또한, 임상시험(feasibility study)도 진행하고 있습니다. 이러한 경험을 통해, 다양한 데이터 분석 경험을 갖추고 논리적으로 설명하는 역량을 기르고자 합니다.

이 역량들이 결국 규제 관련 업무에도 도움이 될 것으로 기대합니다 ( ex. Real world evidence 창출 등)

나의 성장을 가로막는 벽은 무엇인가?

게으름이라고 생각합니다. 공부해야 하는데, 연구해야 하는데 게으를 때가 있습니다. 이는 결국 커뮤니티 활동을 통해 게으름 극복을 하고자 합니다. 가짜연구소 활동을 통해 강제적으로 공부하게 만드는 것입니다.


#가짜연구소#PseudoLab#성장

**Comment by seriousran** [comment]
9기 아카데미에서는 활동을 아래의 3가지 중 하나로 선택해서 진행할 수 있습니다.
##  1. Fundamental Fellowship
[Datacamp](datacamp.com/)의 후원을 받아, Datacamp의 교육 과정을 무료로 함께 수강하며 성장하는 모임입니다. 기초와 입문 과정인 경우 추천하며, 기초를 탄탄히 하고자 할 때에도 큰 도움이 됩니다! 만족도가 매우 높은 플랫폼이고, 인당 $300 가치의 라이센스를 부여하며, 지역적 격차 등 성장의 기회를 얻지 못한 이들을 우선적으로 선정합니다.
## 2. Connected Study (default)
일반적으로 생각하는 스터디 그룹 & 팀 프로젝트입니다. 자율적으로 계획하고 진행하며, 디비전의 코디네이터가 한 기수동안 온보딩, 팀 빌딩 등 전반적인 과정을 도와줍니다. 특별히 1, 3번을 선택하지 않는다면 2번이 default로 설정됩니다.
## 3. United Research
가짜연구소 내에 존재하는 Research 조직에 편입하여 활동할 수 있습니다. Research는 여러 스터디/프로젝트들을 포함하는 조직으로, 리드가 코디네이터보다는 좀 더 분야에 대한 전문 지식과 경험을 갖고 빌더들을 이끌어줍니다. Research의 다른 스터디/프로젝트/빌더/러너와 교류할 수 있으며, 함께 행사를 진행하는 등 더 유기적인 활동을 할 수 있습니다. 새로운 Research를 구성하기 위해서는 1기수 이상을 빌더로 온전히 마쳐야합니다.

리서치 리드: 신진수
키워드: 인과추론과 실험 플랫폼

### Hugging Face KREW
- 리서치 리더: 정우준
- 리서치 빌더: 임지훈
- 리서치 소개: 🤗 Hugging Face 오픈소스에 기여하며 Hugging Face와 AI를 깊이 이해하고자 노력하는 리서치 팀입니다.
- 우리는 다음과 같은 활동을 통해 함께 성장하고 있습니다:
  - Hugging Face 기술 문서를 이해하고 번역하며 AI 지식을 확장합니다.
  - 더 나아가, 직접 기술 문서를 작성하고 기여하며 Hugging Face 오픈소스에 가치를 더합니다.
  - 모델 리소스 컨트리뷰션을 통해 다양한 딥러닝 모델들을 탐구하고, Hugging Face를 활용한 사용 방법을 연구합니다.
  - Hugging Face 코드를 분석하고 직접 코드 기여에 참여하여 개발 경험을 쌓고자 합니다.
  - 이러한 활동을 통해 Hugging Face 한국 커뮤니티 활성화에 기여하고자 합니다. 우리의 경험과 지식을 공유하며 함께 배우고 발전하는 커뮤니티를 만들어 가는 것이 목표입니다.
- 진행 중인 스터디
  - OSSCA 2024에 Hugging Face OSS 팀으로 참가해 멘토링 진행 중.
- 완료한 스터디/프로젝트명: 
  - Hugging Face 온라인 문서 번역(S6)
  - OSSCA 2023 - Hugging Face 한글화 프로젝트(S7)
  - Hugging Face KREW Hackathon 2023: Everyday AI(🤗 Hugging Face 후원)
  - Hugging Face 공식 문서 번역(S8)
- 주요 결과물 소개: 
  - [Transformers 문서 번역](https://huggingface.co/docs/transformers/v4.43.4/ko/index)
  - [Python Hub Library 문서 번역](https://huggingface.co/docs/huggingface_hub/v0.24.5/ko/index)
  - [Hugging Face와 협업해서 Hackathon 개최](https://huggingface.co/collections/pseudolab/hugging-face-krew-hackathon-2023-652bb5066f88e29851af8076)
  - [번역 Glossary 사이트](https://pseudo-lab.github.io/deep-learning-glossary/dictionary/glossary.html)
- 협업/후원 기업: 🤗 Hugging Face
- 기타 추가사항: -

### 추천시스템 리서치
- 리서치 리더: 이경찬
- 리서치 빌더: 이경찬
- 리서치 소개: 지난 1년 반 동안 10편 이상의 추천시스템 주요 논문 리뷰 후 PseudoRec이라는 영화 추천시스템 사이트를 구현하며 개발 경험도 쌓았습니다. Sequential 및 Graph-based 등 최신 추천 모델과 CF, MF 등의 전통적인 추천 기법을 해당사이트에 구현하고 결과를 비교하며 토론합니다. 최근엔 LLM을 활용한 개인화 추천을 구현 중입니다. 또한 ‘월간슈도렉’이라는 이름으로 최신 추천시스템 관련 자료 리뷰, 또는 추천 모델 구현기를 월 1회 정리하며 Build in Public을 수행해 나갑니다.
- 진행중인 스터디/프로젝트명
  - 추천시스템 논문에서 서비스까지 2(S8~ : 여름 계절학기 진행중)
- 완료한 스터디/프로젝트명
  - 추천시스템 주요 논문 리뷰 및 구현(S6)
  - 추천시스템 논문에서 서비스까지 1(S7)
- 주요 결과물 소개: https://www.pseudorec.com/

### PseudoDiffusers
- 리서치 리더: 조상우
- 리서치 빌더: 조상우
- 리서치 소개: PseudoDiffusers 는 2년째 운영되고 있는 조직이며, 최근에는 CVPR 2024 Workshop 에 논문을 게재하기도 하였습니다. 스터디 주제로는 최신 text-to-image generation, text-to-video generation, 그리고 text-to-3d generation 모델들에 대해 논문 및 코드를 이론적으로 접근하며 리뷰합니다.
- 진행 중인 스터디: Text-to-Image Generation (feat.Diffusion) - Season 4 (진행 예정)
- 완료한 스터디/프로젝트명
  - Text-to-Image Generation (feat.Diffusion) - Season 1
  - Text-to-Image Generation (feat.Diffusion) - Season 2
  - Text-to-Image Generation (feat.Diffusion) - Season 3
- 주요 결과물 소개
  - DiffInject: Revisiting Debias via Synthetic Data Generation using Diffusion-based Style Injection (CVPR 2024 Workshop) 논문 게재
  - PseudoDiffusers 깃헙 블로그 운영 (https://github.com/Pseudo-Lab/pseudodiffusers)
협업/후원 기업: -
기타 추가사항: -

### 에이전트 빌더 길드
- 리서치 리더: 정한결
- 리서치 빌더: 정한결 (현재 논의중) 
- 리서치 소개: llm 엔지니어링, 자동화, 에이전트를 주제로 프로덕트화를 지향하는 리서치 팀입니다. 
- 참가자들은 llm 엔지니어링 관련된 논문을 읽어와 조사하고, 이를 자신의 프로덕트에 어떻게 구현했는지 결과를 발표합니다.  (즉, 매주 컨퍼런스식 발표를 하는 셈입니다).
- 궁극적으로 제작자들이 모여서 성장하는 도장,  또 장기적으로 협업할 수 있는 길드를 지향합니다. 
이번 시즌에는 llm agent 제작의 엔지니어링 부분에 대한 위키 핸드북을 낼 예정이고, 디스코드에서 하는 스터디를 외부 플랫폼에서 공개 송출해 볼 생각입니다.
- 진행중인 스터디/프로젝트명: 깃헙에 NLP 잔디심기 S5 
- 완료한 스터디/프로젝트명
  - 깃잔심S1(5기)
  - 깃잔심S2(6기)
  - 깃잔심S3(7기)
  - 깃잔심S4(8기)
- 주요 결과물 소개: (쥬피터 북 페이지 - 만드는 중!) 
- 협업/후원 기업: X, 추가 예정 
- 기타 추가사항: 슈도랩 외부의 그룹들과도 같이 작업하려고 합니다! 생기는 대로 공유할게요

---

1-3 중 희망하는 것이 있다면 이 글에 댓글을 달아주세요! 해당 펠로우쉽 디렉터 혹은 리서치 리드와 협의하도록 하겠습니다!
궁금한 사항이 있다면 편히 [Q&A](https://github.com/Pseudo-Lab/9th-builder/discussions/categories/q-a)를 이용해주세요.

**Reply by shstat1729** [reply]
3번 (Bridging Causal Inference and Real-World Evidence: A Study of FDA and Other Regulatory Guidelines)
리서치 리드: 신진수
키워드: 인과추론과 실험 플랫폼

감사합니다.

**Reply by potea** [reply]
1번 "AI를 잘 활용하는 개발자로 성장하기 #2"
DataCamp 수강을 기반으로 토이프로젝트를 수행합니다.
